api/config/initializers/cors.rb

```diff
@@ -5,12 +5,13 @@
 
 # Read more: https://github.com/cyu/rack-cors
 
-# Rails.application.config.middleware.insert_before 0, Rack::Cors do
-#   allow do
-#     origins "example.com"
-#
-#     resource "*",
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins "*" #TODO: for prod change to domain of frontend app
```

Comment:
This could probably be something that you make a config variable in the environment files. 

Like in api/config/environments/development.rb you could say

`config.allowed_cors_origins = "*"`

and then in api/config/environments/production.rb you could do 

`config.allowed_cors_origins = ["TODO.com"]` and include the comment about prod changes there. That way, once you have the frontend domain you won't have any local dev issues with it trying to use the production frontend